src/app/components/Fordeling/__tests__/Fordelingskomponent.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { axe } from 'jest-axe';
+import Fordelingskomponent from "../Fordelingskomponent";
+import navColors from "../../../../styles/designSystemColors";
+import React, {useState} from 'react';
```

Comment:
`useState` brukes ikke.
```suggestion
import React from 'react';
```